src/_scss/pages/interactiveDataSources/index.scss

```diff
@@ -36,14 +36,10 @@
   .main-content {
     @include display(flex);
     width: 100%;
-
-    @media(min-width: $large-screen) {
-      width: 75%;
-    }
     @include flex-grow(1);
     @include flex-wrap(wrap);
-    max-width: rem(1600);
-    margin: auto;
+    max-width: rem(1400);
+    margin: 0 rem(32);
```

Comment:
I think the margin on line 42 can be removed. Removing it in webtools doesn't change its behavior.